contrib/transformations/lib/transformations/with_indifferent_access.rb

```diff
@@ -8,7 +8,8 @@ def dump(item)
         event_id:   item.event_id,
         metadata:   ActiveSupport::HashWithIndifferentAccess.new(item.metadata).deep_symbolize_keys,
         data:       ActiveSupport::HashWithIndifferentAccess.new(item.data).deep_symbolize_keys,
-        event_type: item.event_type
+        event_type: item.event_type,
```

Comment:
We should exclude this change in contrib project. It depends on already released RubyEventStore gem version and this will break the tests as timestamp changes has not yet been released. Also IMO we should avoid mixing contrib projects changes into PR with main gems code.